src/Functions/minSampleSize.cpp

```diff
@@ -0,0 +1,247 @@
+            {
+                "minimum_sample_size",
+                "detect_range_lower",
+                "detect_range_upper",
+            };
+
+            return std::make_shared<DataTypeTuple>(
+                std::move(types),
+                std::move(names)
+            );
+        }
+
+        DataTypePtr getReturnTypeImpl(const DataTypes & arguments) const override
+        {
+            Impl::validateArguments(arguments);
+            return getReturnType();
+        }
+
+        ColumnPtr executeImpl(const ColumnsWithTypeAndName & arguments, const DataTypePtr &, size_t input_rows_count) const override
+        {
+            return Impl::execute(arguments, input_rows_count);
+        }
+    };
+
+
+    static bool isBetweenZeroAndOne(Float64 v)
+    {
+        return v >= 0.0 && v <= 1.0 && fabs(v - 0.0) >= DBL_EPSILON && fabs(v - 1.0) >= DBL_EPSILON;
+    }
+
+
+    struct ContinousImpl
+    {
+        static constexpr auto name = "minSampleSizeContinous";
+        static constexpr size_t num_args = 5;
+
+        static void validateArguments(const DataTypes & arguments)
+        {
+            for (size_t i = 0; i < 1; ++i)
+            {
+                if (!isNativeNumber(arguments[i]))
+                {
+                    throw Exception(ErrorCodes::ILLEGAL_TYPE_OF_ARGUMENT, "The {}th Argument of function {} must be a native number.", i + 1, name);
+                }
+            }
+
+            for (size_t i = 2; i < arguments.size(); ++i)
+            {
+                if (!isFloat(arguments[i]))
+                {
+                    throw Exception(ErrorCodes::ILLEGAL_TYPE_OF_ARGUMENT, "The {}th Argument of function {} must be a float.", i + 1, name);
+                }
+            }
+        }
+
+        static ColumnPtr execute(const ColumnsWithTypeAndName & arguments, size_t input_rows_count)
+        {
+            const IColumn & col_baseline = *arguments[0].column;
+            const IColumn & col_sigma = *arguments[1].column;
+            const IColumn & col_mde = *arguments[2].column;
+            const IColumn & col_power = *arguments[3].column;
+            const IColumn & col_alpha = *arguments[4].column;
+
+            auto res_min_sample_size = ColumnFloat64::create();
+            auto & data_min_sample_size = res_min_sample_size->getData();
+            data_min_sample_size.reserve(input_rows_count);
+
+            auto res_detect_lower = ColumnFloat64::create();
+            auto & data_detect_lower = res_detect_lower->getData();
+            data_detect_lower.reserve(input_rows_count);
+
+            auto res_detect_upper = ColumnFloat64::create();
+            auto & data_detect_upper = res_detect_upper->getData();
+            data_detect_upper.reserve(input_rows_count);
+
+            for (size_t row_num = 0; row_num < input_rows_count; ++row_num)
+            {
+                /// Mean of control-metric
+                Float64 baseline = col_baseline.getFloat64(row_num);
```

Comment:
```c++
ColumnVector
Float64 getFloat64(size_t n) const override
{
    if constexpr (is_arithmetic_v<T>)
        return static_cast<Float64>(data[n]);
    else
        throw Exception(ErrorCodes::NOT_IMPLEMENTED, "Cannot get the value of {} as Float64", TypeName<T>);
}
```
it too heavy method, please use `getData` from Column to get array of Float64.

Reply:
I fixed it to use `getData`.